src/json.ts

```diff
@@ -36,7 +36,7 @@ function intFromEnv(envName: string): number | undefined {
     return
   }
   const parsed = parseInt(value)
-  if (parsed === -1) {
+  if (isNaN(parsed)) {
```

Comment:
isnt this taken care of in parseInt ? what about infinity / minus infinity (checkFinite)

Reply:
This is the standard parseInt that will return NaN if the string is not a number.